index.js

```diff
@@ -129,15 +129,17 @@ var PgDriver = Base.extend({
     if (typeof options === 'object') {
       if (typeof options.database === 'string') {
         this.log.info(
-          'Ignore database option, not available with postgres. Use schema instead!'
+            'Ignore database option, not available with postgres. Use schema instead!'
         );
+      }
+      if (typeof options.schema === 'string') {
         this.runSql(
-          util.format('SET search_path TO `%s`', options.database),
```

Comment:
this will certainly break a lot of dbs connected using the pg wire protocol, so that can't just be overwritten here (also in pg dbs exist though